From: John Paul Adrian Glaubitz <glaubitz@physik.fu-berlin.de>
Date: Sun, 15 Apr 2018 08:49:43 +0200
Subject: Use -msse{2} -fpmath C-/CXXFLAGS only on x86_64 platforms

The options -msse -msse2 -mfpmath=sse are only available on AMD64 based
platforms.
---
 build/unix/mozconfig.linux32 | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/build/unix/mozconfig.linux32 b/build/unix/mozconfig.linux32
index b9ac630..cf68b49 100644
--- a/build/unix/mozconfig.linux32
+++ b/build/unix/mozconfig.linux32
@@ -2,12 +2,11 @@
 
 export MOZ_LINUX_32_SSE2_STARTUP_ERROR=1
 
-CFLAGS="$CFLAGS -msse -msse2 -mfpmath=sse"
-CXXFLAGS="$CXXFLAGS -msse -msse2 -mfpmath=sse"
-
 if test `uname -m` = "x86_64"; then
   CC="$CC -m32 -march=pentium-m"
   CXX="$CXX -m32 -march=pentium-m"
   ac_add_options --target=i686-pc-linux
   ac_add_options --host=i686-pc-linux
+  CFLAGS="$CFLAGS -msse -msse2 -mfpmath=sse"
+  CXXFLAGS="$CXXFLAGS -msse -msse2 -mfpmath=sse"
 fi
